src/content/docs/more-integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry.mdx

```diff
@@ -1,31 +1,150 @@
+You can configure the OpenTelemetry bridge to forward Micrometer data to New Relic. The OpenTelemetry Micrometer bridge directs Micrometer to record telemetry to OpenTelemetry, which you can then export to New Relic with the OpenTelemtry SDK. 
 
-New Relic's Micrometer metrics registry sends your Micrometer telemetry data to your New Relic account. Don't have one? [Sign up for free](https://newrelic.com/signup). No credit card required.
+This doc walks you through procedures to forward Micrometer data with an example Spring app. These are generalized steps that you may want to update to fit your specific environment. 
 
-Resources for our Micrometer integration:
+## Compatibility and requirements [#requirements]
 
-* [README](https://github.com/newrelic/micrometer-registry-newrelic)
-* [Integration specs](https://github.com/newrelic/newrelic-exporter-specs/tree/master/micrometer)
+Before you get started, we recommend that review the requirements outlined in the [Getting started](https://github.com/open-telemetry/opentelemetry-java-instrumentation?tab=readme-ov-file#getting-started) section of the OpenTelemetry Java instrumentation repo. Once you've reviewed this section, return to this doc.
 
-<InstallFeedback />
+At minimum, you should:
 
-## Find your data
+* Have a Springboot app with Micrometer enabled. Your Springboot app should be instrumented with Micrometer out of the box.
+* Ensure your OpenTelemetry Java agent is the latest version.
 
-To find your data, go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > All entities**</DoNotTranslate>. Search for your service by name. We provide an out-of-the-box overview that includes charts for some of the most useful metrics provided by Micrometer Spring Boot instrumentation.
+## Forward Micrometer data to New Relic [#forward-data-OpenTelemetry]
 
-You can use the out-of-the-box overview as a starting point to build your own dashboards. Click on the `...` icon in a chart to add that chart to a dashboard or view the query behind the chart to modify it and it add it to your dashboard.
+<Steps>
+    <Step>
 
-For more on how to query your data once it's in New Relic, see [Query your data](/docs/using-new-relic/data/understand-data/query-new-relic-data) and [Introduction to NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql).
+### Add OpenTelemetry Micrometer instrumentation
+
+Add OpenTelemetry Micrometer instrumentation to the alpha modules section of your `build.gradle` file:
+
+    ```java
+    //Alpha modules
+    implementation 'io.opentelemetry.instrumentation:opentelemetry-micrometer-1.5'
+    ```
+
+    </Step>
+    <Step>
+
+### Add OpenTelemetry API and SDK dependencies
+        
+In the `dependencies()` argument, add the OpenTelemetry API and SDK:
+
+    ```java
+    dependencies {
+        implementation 'io.opentelemetry:opentelemetry-api'
```

Comment:
```suggestion
```
This isn't really necessary. I'll update the example code to reflect this and other recommendations I'm making here.